.circleci/config.yml

```diff
@@ -0,0 +1,295 @@
+version: 2.1
+
+
+orbs:
+  mim:
+    commands:
+      install_erlang:
+        parameters:
+          otp:
+            type: string
+            description: Version of the Erlang package to install
+        steps:
+        - run:
+            name: Install Erlang
+            command: |
+              wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
+              sudo dpkg -i erlang-solutions_1.0_all.deb
+              sudo apt-get update
+              sudo apt-get install -y esl-erlang=<<parameters.otp>>
+      fetch_packages:
+        steps:
+        - run:
+            name: Install basic packages
+            command: |
+              sudo apt install unixodbc-dev
+              sudo apt install unixodbc
+              sudo apt install tdsodbc
+    jobs:
+      build_and_small_tests:
+        parallelism: 1
+        machine:
+          docker_layer_caching: false
+        parameters:
+          otp:
+            type: string
+            description: Version of the Erlang package to install
+          build_releases:
+            type: string
+            description: Flag to build release or not
+            default: "1"
+
+        environment:
+          SKIP_RELEASE: 1
+          SKIP_COV: 0
+          PRESET: small_tests
+        working_directory: ~/app
+
+        steps:
+          - checkout
+          - install_erlang:
+              otp: <<parameters.otp>>
+          - run:
+              name: Prepare for cache
+              command: |
+                ./tools/circleci-otp-to-str.sh <<parameters.otp>> > otp_version
+          - restore_cache:
+              key: build-cache-{{ .Branch }}-{{ .Revision }}--{{ checksum "otp_version" }}
+          - restore_cache:
+              key: deps-cache--{{ checksum "rebar.lock" }}--{{ checksum "big_tests/rebar.lock" }}--{{ checksum "otp_version" }}
+          - fetch_packages
+          - run:
+              name: Get deps
+              command: |
+                sudo apt install python-pip
```

Comment:
Why isn't that part of `fetch_packages` orb? Also, all comments I've posted for that orb, also apply here.

Reply:
Orbs are not enabled yet for the Mongooseim project, so for now I've manually installed packages.